File: src/main/kotlin/kindergarten/web/entity/KgClassroom.java
package main.kotlin.kindergarten.web.entity;
import java.io.Serializable;
import java.math.*;
import java.util.Date;
import java.sql.Timestamp;

/*
* 
* gen by beetlsql 2017-07-03
*/
public class KgClassroom   implements Serializable{
	private Integer id ;
	private Integer chirldCount ;
	private Integer schoolId ;
	private String classroomImage ;
	private String title ;
	
	public KgClassroom() {
	}
	
	public Integer getId(){
		return  id;
	}
	public void setId(Integer id ){
		this.id = id;
	}
	
	public Integer getChirldCount(){
		return  chirldCount;
	}
	public void setChirldCount(Integer chirldCount ){
		this.chirldCount = chirldCount;
	}
	
	public Integer getSchoolId(){
		return  schoolId;
	}
	public void setSchoolId(Integer schoolId ){
		this.schoolId = schoolId;
	}
	
	public String getClassroomImage(){
		return  classroomImage;
	}
	public void setClassroomImage(String classroomImage ){
		this.classroomImage = classroomImage;
	}
	
	public String getTitle(){
		return  title;
	}
	public void setTitle(String title ){
		this.title = title;
	}
	
	
	

}

File: src/main/kotlin/kindergarten/web/entity/KgClassroom.kt
package kindergarten.web.entity

import com.fasterxml.jackson.annotation.JsonIgnore
import kindergarten.custom.CustomTailBean
import java.io.Serializable


class KgClassroom : Serializable, CustomTailBean() {
    @JsonIgnore
    var id: Int? = null
    var childCount: Int? = null
    //是否是走廊或者通道 走廊通道是1 教室是0
    var isCorridor: Int? = null
    @JsonIgnore
    var schoolId: Int? = null
    var classroomImage: String? = null
    var showName: String? = null
    //0是摄像头关闭 1是开始
    var isUnWatch: Int? = null
    var unWatch: Int = 0
    var synopsis: String? = null

}